### ISSUE TITLE
Improve F-Droid listing

### ISSUE THREAD

**Issue by mueller-ma** [ligi/SCR#9]
I've seen this app on F-Droid, but it lacks screenshots, changelog and a translatable description there. It's pretty easy for you to add these information. Please have a look at [this example](https://github.com/Freeyourgadget/Gadgetbridge/tree/master/fastlane/metadata/android) or at the [F-Droid docs](https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/#in-the-apps-source-repository) or [fastlane docs](https://github.com/fastlane/fastlane/blob/2.28.7/supply/README.md#images-and-screenshots).
After you have commited these files to your repo and tagged a new version, you need to remove the `Description` and `Summary` from the metadata file in the [F-Droid Data repo](https://gitlab.com/fdroid/fdroiddata).

**Comment by ligi** [comment]
Thanks - but I do not think I will do it for this app - only used for the congress - might do it for other apps of mine on f-droid that are used all year

**Comment by mueller-ma** [comment]
So this app was only used for one congress? Maybe it should be archived in F-Droid, when it isn't usefull anymore.

**Comment by ligi** [comment]
Yes this is only used for preparation for the chaos congress. What advantages would archiving bring? I don't have a strong feeling but I think it should stay as people might want to try something out. I do not really see an advantage of removing something. You also have apps on play that are just used for one event.

**Comment by ligi** [comment]
let's keep it open - will update it most likely for 35c3 and this is a good reminder then to add the screenshots and stuff

**Comment by mueller-ma** [comment]
> will update it most likely for 35c3

Ok, in this case it's absolute valid to keep the app in F-Droid.
